analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.cs

```diff
@@ -839,7 +839,7 @@ public void ReferenceEqualsBool()
         {
             bool a, b;
             a = b = true;
-            if (object.ReferenceEquals(a, b)) { }   // FN
+            if (object.ReferenceEquals(a, b)) { }   // Noncompliant {{Change this condition so that it does not always evaluate to 'True'.}}
```

Comment:
This always evaluates to false. The problem. You will need to adjust the `ReferenceEquals` to fix this.

Reply:
🤦 Indeed. This should not even raise (The pr only deals with `Equals`(.